manifests/install.pp

```diff
@@ -4,6 +4,6 @@
   include varnish::params
 
   package { $varnish::params::package_name:
-    ensure => present,
+    ensure => $varnish::params::package_ensure,
```

Comment:
This should just be `ensure => $varnish::package_ensure` so we can specify version from `class { 'varnish': `.

:)